README.md

```diff
@@ -1505,6 +1578,40 @@ There is an optional field called `isHidden` available on both buttons. If set t
 the `display-none` class will be added to the widget's classes resulting in the button being hidden
 when the page loads.
 
+### Screen with One Input
+
+Screens with a single input are tricky because all inputs need a label, but for this screen pattern
+the label is the header of the page. To handle this, we have a special fragment called `screenWithOneInput`.
```

Comment:
I think you should clarify _why_ they need a label. It's for accessibility reasons, right?  So a screen reader can pars the page correctly?